src/container/chartStore.ts

```diff
@@ -0,0 +1,45 @@
+import { BaseStore } from "../app/stores/base";
```

Comment:
Perhaps it's better not to import from ../app, doing so may increase the size of the exported visual with unnecessary code (possibly the whole React). We can refactor the code and move common stuff to a separate folder.

Reply:
Whoops, I was actually going to pull that into that out into the 'core' module, since both the `app` and the `container` module use it, but then I forgot, thanks.